grass7/raster/r.hypso/r.hypso.py

```diff
@@ -126,6 +123,10 @@ def findint(kl,f):
     return z
 
 def plotImage(x,y,image,type,xlabel,ylabel,title):
+    import matplotlib #required by windows
+    matplotlib.use('wx') #required by windows
```

Comment:
```suggestion
    matplotlib.use('wxAGG') #required by windows
```
...as in grass7/display/d.vect.colbp/d.vect.colbp.py (?)